app/controllers/api/v1/book_suggestions_controller.rb

```diff
@@ -0,0 +1,30 @@
+module Api
+  module V1
+    class BookSuggestionsController < Api::ApiController
+      skip_before_action :authenticate_api_v1_user!, only: [:create]
+
+      def create
+        @book_suggestion = BookSuggestion.new(book_suggestion_params)
+
+        if current_api_v1_user.present?
```

Comment:
you could use do this as a `before_action` on the `ApiController`

Reply:
you mean set the current user in the new instance?